app/assets/stylesheets/components/_background_color.scss

```diff
@@ -0,0 +1,4 @@
+.wrapper{
```

Comment:
@Jadiman17 you have this same bit of scss applied for the same .wrapper class in the _devise_user.scss file.  Are they both necessary?  I ask because they are named the same and could create conflict